File: 2542-average-value-of-even-numbers-that-are-divisible-by-three/2542-average-value-of-even-numbers-that-are-divisible-by-three.py
class Solution:
    def averageValue(self, nums: List[int]) -> int:
        c=0

        k=0
        for i in nums:
            if(i%6==0):
                k+=i
                c+=1
        if(c==0):
            return 0
        return k//c
        

File: 2542-average-value-of-even-numbers-that-are-divisible-by-three/2542-average-value-of-even-numbers-that-are-divisible-by-three.cpp
class Solution {
public:
    int averageValue(vector<int>& nums) {
        int c=0,k=0;
        for(int it:nums){
            if(it%6==0){
                k+=it;
                c++;
            }
        }
        if(c==0){
            return 0;
        }
        return k/c;
        
    }
};